client/my-sites/posts/main.jsx

```diff
@@ -55,6 +56,7 @@ class PostsMain extends React.Component {
 	render() {
 		const { author, category, search, siteId, statusSlug, tag, translate } = this.props;
 		const status = mapPostStatus( statusSlug );
+		const allSites = siteId ? 1 : 2;
```

Comment:
Same as above - I'm guessing you didn't mean to commit this line and it was just for testing.